09_2024_scikit_learn_priorities.md

```diff
@@ -0,0 +1,260 @@
+6 months.
+
+As we reflect on our progress and look ahead, it is important to reiterate the key
+considerations that guide our decision-making process:
+
+* Identifying big picture goals and themes that improve user experience and provide
+  tools for the entire machine learning pipeline, from training and exploration to
+  production.
+* Insights from community surveys conducted in 2023 to understand core contributors'
+  concerns and priorities. Additionally, we will soon incorporate feedback from the
+  ongoing [user `scikit-learn` survey](https://forms.gle/zUXvWjGUN1nWhJ2V6) to further
+  align our efforts with user needs and expectations.
+* Commitment to revisiting our list of priorities approximately twice a year.
+
+Let's review our achievements and set the stage for our upcoming focus areas.
+
+## Retrospective on the past 6 months
+
+This section summarizes the progress of Probabl's work on the different supported
+open-source projects.
+
+### [`scikit-learn`](https://github.com/scikit-learn/scikit-learn)
+
+In a previous [blog
+post](https://papers.probabl.ai/scikit-learns-priorities-at-probabl), we outlined
+Probabl's priorities for `scikit-learn`. In this section, we report on the progress of
+these priorities.
+
+- Short-term priorities
+
+   1. Website Migration: We successfully migrated the `scikit-learn` website to the new
+      PyData theme-based website. This modernization effort improves the user
+      experience and aligns our documentation with the broader PyData ecosystem.
+   1. SLEP6 - Metadata Routing: We completed adapting most estimators to handle
+      metadata routing as specified in
+      [SLEP6](https://scikit-learn-enhancement-proposals.readthedocs.io/en/latest/slep006/proposal.html).
+      This new API consistently handles metadata (e.g., `sample_weight`, `groups`, etc.)
+      for almost all estimators. It allows for fewer bugs and enables new
+   use cases that were not possible before.
+
+Both of these projects represent major milestones for `scikit-learn`. In our current
+round of priorities, we outline follow-up work for both the website and metadata
+routing to further improve and expand on these changes.
+
+-  Mid-term priorities
+
+   1. Implementation of a new `scikit-learn` estimator, called
+      `TunedThresholdClassifierCV`, allowing optimization of the operational decision.
+   1. Progress on the Array-API adoption by making it possible to train and test a
+      pipeline containing a `PCA` and `Ridge` estimator trained on `torch` tensors
+      located on GPU.
+   1. Some work to improve the developer experience by improving the developer tools
+      available in `scikit-learn` and made available for third-party projects.
+
+- Long-term priorities
+
+   1. General maintenance of the project: We reviewed a substantial number of pull
+      requests and issues opened by external contributors.
+   1. Python 3.13 support: We worked on supporting Python 3.13 free-threaded mode and
```

Comment:
Was it together with Nathan from Quansight?

Reply:
Indeed, it started with this [issue](https://github.com/scikit-learn/scikit-learn/issues/28978) and now Nathan is also involved in this [thread](https://github.com/scikit-learn/scikit-learn/issues/30007) regarding the stress-testing phase. I think Loïc is the person the most involved on this topic from our side.

Do you want me to have an explicit acknowledgement to the involvement of Nathan on this topic?